### ISSUE TITLE
Run several testsuites

### ISSUE THREAD

**Issue by kindundtraum** [paratestphp/paratest#347]
Here is the definition of testsuites in my phpunit.xml:

<?xml version="1.0" encoding="UTF-8"?>

    <testsuites>
        <testsuite name="cdc">
            <directory>tests/cdc/</directory>
        </testsuite>
        <testsuite name="unit">
            <directory>tests/unit/</directory>
        </testsuite>
        <testsuite name="integration">
            <directory>tests/integration/</directory>
        </testsuite>
        <testsuite name="ci">
            <directory>tests/unit/</directory>
            <directory>tests/integration/</directory>
        </testsuite>
    </testsuites>

I run all this using this command:

> php ./vendor/bin/paratest --testsuite=ci

As you can see I have to define new testsuite called 'ci' to run unit & integration but not cdc tests. It would be great if I could launch several testsuites without defining new one, just like in phpunit:

> php ./vendor/bin/phpunit --testsuite=unit,integration

**Comment by paslandau** [comment]
Just ran into the same problem. Multiple testuites separated by "," are supported by phpunit since 6.0.0 (see https://github.com/sebastianbergmann/phpunit/commit/80754cf323fe96003a2567f5e57404fddecff3bf ) so I'd argue that is a reasonable assumption for a paratest user to get the same behavior.

Had a quick look through the corresponding code and the change itself seems rather easy:
- `ParaTest\Runners\PHPUnit\Runner::getRunnerOptions` should simply explode the given testsuite on ","
- `ParaTest\Runners\PHPUnit\Suiteloader::load()` already has the required functionality, because it "falls back" to "all defined suites" if none is given, see
```php
        if ($path) {
            $testFileLoader = new TestFileLoader($this->options);
            $this->files = array_merge($this->files, $testFileLoader->loadPath($path));
        } elseif (isset($this->options->testsuite) && $this->options->testsuite) {
            foreach ($configuration->getSuiteByName($this->options->testsuite) as $suite) {
                foreach ($suite as $suitePath) {
                    $testFileLoader = new TestFileLoader($this->options);
                    $this->files = array_merge($this->files, $testFileLoader->loadSuitePath($suitePath));
                }
            }
        } elseif ($suites = $configuration->getSuites()) {
            foreach ($suites as $suite) {
                foreach ($suite as $suitePath) {
                    $testFileLoader = new TestFileLoader($this->options);
                    $this->files = array_merge($this->files, $testFileLoader->loadSuitePath($suitePath));
                }
            }
        }
```

So we can easily change `$suites = $configuration->getSuites()` to filter `suites` if `$this->options->testsuite` contains a testsuite.
```php
        if ($path) {
            $testFileLoader = new TestFileLoader($this->options);
            $this->files = array_merge($this->files, $testFileLoader->loadPath($path));
        } elseif ($suites = $configuration->getSuites()) {
            if (isset($this->options->testsuite) && $this->options->testsuite) {
              // filter $suites.
            }
            foreach ($suites as $suite) {
                foreach ($suite as $suitePath) {
                    $testFileLoader = new TestFileLoader($this->options);
                    $this->files = array_merge($this->files, $testFileLoader->loadSuitePath($suitePath));
                }
            }
        }
```
@andreasschroth - are you open to that? I'd prepare a PR if you're fine with that.

Cheers
Pascal

**Comment by paslandau** [comment]
Do you have any crontibution guidelines (e.g. easy way to run the tests, run the style check, ...)?

Didn't see a CONTRIBUTING.md, `composer run-script -l` is empty and I don't see a Makefile (or the likes) - so I would now stick to your `.travis.yml` install targets?

**Comment by andreasschroth** [comment]
@paslandau Me and a few others were volunteering as maintainers for this project and the original maintainer left / is not interested in this anymore. So yes, I would say you can ignore this for now and still submit a PR.

**Comment by paslandau** [comment]
@andreasschroth kk, thx for getting back - will prepare something tomorrow.

Btw. didn't want to come off "demanding", sorry if my last comment gave that impression.

Thx for maintaining :)

**Comment by paslandau** [comment]
FYI: Tests failed because `pdo_sqlite` extension was missing on my system. I added `markTestSkipped`for those tests if the extension is missing.

Without extenstion
```
root@db51901fe7a6:/codebase/paratest# composer test
Do not run Composer as root/super user! See https://getcomposer.org/root for details
> phpunit
PHPUnit 7.5.2 by Sebastian Bergmann and contributors.

.........S.....................................................  63 / 252 ( 25%)
............................................................... 126 / 252 ( 50%)
.....................................SSSSSSSSSS................ 189 / 252 ( 75%)
....S......S.............................SS............SSSS.... 252 / 252 (100%)


Time: 47.23 seconds, Memory: 8.00MB

OK, but incomplete, skipped, or risky tests!
Tests: 252, Assertions: 463, Skipped: 19.
```

With extension
```
root@db51901fe7a6:/codebase/paratest# apt-get install php7.2-sqlite
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'php7.2-sqlite3' for regex 'php7.2-sqlite'
The following NEW packages will be installed:
  php7.2-sqlite3
0 upgraded, 1 newly installed, 0 to remove and 53 not upgraded.
Need to get 24.2 kB of archives.
After this operation, 112 kB of additional disk space will be used.
Get:1 http://ppa.launchpad.net/ondrej/php/ubuntu xenial/main amd64 php7.2-sqlite3 amd64 7.2.14-1+ubuntu16.04.1+deb.sury.org+1 [24.2 kB]
Fetched 24.2 kB in 0s (237 kB/s)
Selecting previously unselected package php7.2-sqlite3.
(Reading database ... 16537 files and directories currently installed.)
Preparing to unpack .../php7.2-sqlite3_7.2.14-1+ubuntu16.04.1+deb.sury.org+1_amd64.deb ...
Unpacking php7.2-sqlite3 (7.2.14-1+ubuntu16.04.1+deb.sury.org+1) ...
Setting up php7.2-sqlite3 (7.2.14-1+ubuntu16.04.1+deb.sury.org+1) ...

Creating config file /etc/php/7.2/mods-available/sqlite3.ini with new version

Creating config file /etc/php/7.2/mods-available/pdo_sqlite.ini with new version
root@db51901fe7a6:/codebase/paratest# php -m | grep sqlite
pdo_sqlite
sqlite3
root@db51901fe7a6:/codebase/paratest# composer test
Do not run Composer as root/super user! See https://getcomposer.org/root for details
> phpunit
PHPUnit 7.5.2 by Sebastian Bergmann and contributors.

.........S.....................................................  63 / 252 ( 25%)
............................................................... 126 / 252 ( 50%)
.....................................SSSSSSSSSS................ 189 / 252 ( 75%)
.........................................SS.................... 252 / 252 (100%)


Time: 51.57 seconds, Memory: 8.00MB

OK, but incomplete, skipped, or risky tests!
Tests: 252, Assertions: 487, Skipped: 13.
```

**Comment by andreasschroth** [comment]
No worries, you didn't come off demanding at all - your questions were all absolutely legitimate!

Thank you a lot for the awesome PRs, I'll have a look at them.

**Comment by andreasschroth** [comment]
Feature available thanks to @paslandau in PR #359 .
